### ISSUE TITLE
[a11y]: DropDown component does not announce the options in JAWS

### ISSUE THREAD

**Issue by VishwaKumar** [carbon-design-system/carbon#19387]
### Package

@carbon/react

### Browser

Firefox

### Operating System

Windows

### Package version

1.81.0

### React version

17.0.2

### Automated testing tool and ruleset

JAWS

### Assistive technology

JAWS

### Description
1]
Issue description: SR is not announcing the list of options

Recommended fix: SR must announce the list of options in dropdown menu if user navigates through it

User impact: If the screen reader doesn't announce the menu options when navigating, users with visual impairments may be unable to understand or interact with the menu, leading to frustration and reduced accessibility.

2]
Issue description: When one option is selected in the combobox screen reader is announcing 2nd unselected option in the list of combobox.

Recommended fix: Screen reader should announce the list of combobox selected state.

User impact: if proper selection is not announced along with controls information, it can make it difficult for users who rely on screen readers or other assistive technologies to understand the context and purpose of the control.

WCAG Mapping: 4.1.2 Name, Role, Value

Level of Compliance: A

Severity: Medium

Test tool: JAWS_Firefox

### WCAG 2.1 Violation

WCAG Mapping: 4.1.2 Name, Role, Value

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-dropdown--default

### Steps to reproduce

Navigate to https://react.carbondesignsystem.com/?path=/story/components-dropdown--default
Use JAWS to navigate to dropdown options and observe it does not announce the options

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by skcele** [comment]
Hey there! I have also been seeing this accessibility issue, and it looks like it's affecting IBM Cloud teams pretty heavily. 11 teams have recently had an issue opened for the Dropdown screenreader bug showing up in their product. Happy to provide more info if it would be helpful!

**Comment by mimaishel** [comment]
@skcele has this been resolved in a carbon version. Or still a persistent issue?

This https://github.com/carbon-design-system/carbon/issues/19214#issuecomment-2826636361 mentions `carbon/react` `1.81.0` solving the issue.

How do you recommend to proceed forward in an issue opened with another service concerning this? Move it here? Thanks!

**Comment by skcele** [comment]
Hey @mimaishel, thanks for letting me know! I didn't see anything in the issue about it being resolved. I'll let the Cloud teams know to adopt that version!

**Comment by mimaishel** [comment]
@skcele was getting conflicting local results when testing the current installed version of carbon react (`v1.79.0`) vs. the `v1.81.0`. So was not able to validate that the `v1.81.0` indeed fixes the issue. Any leads or recommended steps would be appreciated. Thanks!

**Comment by tay1orjones** [comment]
We're hearing this [impacting](https://ibm-studios.slack.com/archives/C2K6RFJ1G/p1759177318051979) more teams. We'll try to get this tackled soon.

**Comment by heloiselui** [comment]
It was tested by our accessibility team (Chrome and Firefox) in `@carbon/react v1.95` and confirmed that the issue no longer exists. Dropdown and Combobox behave as expected in these scenarios. I will be closing this issue. If it happens again, we can open a new one or reopen this one.
